app/views/self_service/advisers/_advisers_table.html.erb

```diff
@@ -22,6 +23,9 @@
               <%= link_to adviser.name, edit_self_service_firm_adviser_path(adviser.firm, adviser), class: 't-edit-link' %>
             </td>
             <td class="t-adviser-postcode" data-dough-filterable><%= adviser.postcode %></td>
+            <td>
+              <%= link_to 'Remove from Directory', self_service_firm_adviser_path(firm, adviser), method: :delete, data: {confirm: 'Are you sure?'}, :class => 'button button--small t-remove-button' %>
```

Comment:
There is some jquery schennanigans going on that stops the :delete aspect getting picked up in the frontend. Spent a good while trying to avoid this, however all the advice I can find says to use jquery-ujs.

This is also going to be needed in the next few days when we start deleting firms too.